### PR TITLE
Signup login

### PR DESCRIPTION
**Pull request by ewynx**
The initial screen is now a decision screen which leads to either signup or login.

This is very minimal and the signup screen lacks some info about the privacy, because I imagine people can worry about that and it is a positive point of the app!

After the Signup screen I want to add a little introduction - now it just goes into the questions right away, but I'll make a separate branch for that.

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -66,5 +66,10 @@
   "check all": "Check all that apply",
   "User settings": "User settings",
   "Notification settings": "Notification settings",
-  "Logout": "Log out"
+  "Logout": "Log out",
+  "Create account": "Create account",
+  "go back": "Go back",
+  "Sign up": "Sign up",
+  "enter name": "Please enter your name",
+  "name": "Name"
```
Comment (peterbertel) [inline i18n/en.json:74]: Do you think we need to make similar edits to `es.json` and `nl.json`? I haven't tested running the app with those other languages, so I'm not sure if the app will crash if those values are missing.

**Comment by peterbertel** [comment]
@ewynx I made a couple changes that I think improve the logout experience. Before, the app would take the user to the `login-page-2` screen when clicking the `Log out` button in the settings page. From the `login-page-2` screen, the user was able to click the `Go back` button and navigate back to the app settings page, presumably logging the user back in. To me, I think it feels confusing to be able to log back in by clicking the `Go back` button after logging out. To simplify things, I changed the `Log out` button to send the user to the `login-page-1` screen, which I renamed as the `splash-screen`. Let me know what you think.

File: i18n/en.json (hunk 1 above)
Comment (ewynx) [reply i18n/en.json:74]: You're very correct! Thanks

**Comment by ewynx** [comment]
@peterbertel I like it. On the Log in/Create account screen (that you named splash screen) I disabled the back button, which should add on to that more logical flow for logging out/logging in. So when you're logged out you go to the splash screen and can't get back in the app by clicking on the back button (Android).

**Comment by peterbertel** [comment]
@ewynx Good call disabling the back button for Android. I think the changes are ready to merge!

Feel free to merge when you are ready.

**Review by peterbertel — APPROVED** [review]

**Pull request merged**